<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="makeButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>210</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Make Drink</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>iCub Bar Menu - Please choose your drink</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>211</width>
      <height>191</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Dr. Funk</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zombie</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pina Colada</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pi Yi</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Three Dots and a Dash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Royal Daiquiri</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>BeachComber's gold</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Never Say Die</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Test Pilot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mai Tai</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Deep Sea Driver</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jet Pilot</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Martinique Milk Punch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rum Julep</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Shark Bite</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Strawberry Daiquiri</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mystery Drink</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oh So Deadly</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelImg">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Emergency Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
